README.md

```diff
@@ -1,16 +1,16 @@
 ### Hi there 👋
+I've been working with software development since 2006, i have a degree in Computer Science from UNIVALI and large experience in the hospitality industry
```

Comment:
```suggestion
My name is Augusto Duart Fontana, and i'm from Brazil. I've been working with software development since 2006. I love to code and develop systems, it's something that comes naturally to me and i enjoy doing. 

Always looking to extract the maximum quality in each code line. Clean code for life!!
```